### DISCUSSION TITLE
Extra placeholders

### DISCUSSION THREAD

**Discussion by lloydbayley** [Blackymas/NSPanel_HA_Blueprint#832] [General]
Hi All,

It's not a bug and not quite a feature request if I've missed it, so I'm putting this in general. 
Just wondering if I've missed something...It would be ideal to have multiple placeholders under the weather icon so I could have Outdoor Temperature and Outdoor Humidity one under another...something like the following:

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/21309308/08aec1bd-177e-4452-9975-e1f9d478ba03)

Whilst I understand that things have moved to the right hand side, it would be great to have the outdoor stuff all on the left under the icon.

I've seen this in other people's config shots. Is this something I've missed or have they edited their .tft etc files to allow it?
Would it be possible to allow both the outdoor temp and the humidity (or another HA sensor) on the left hand side?

Cheers,

Lloyd.

**Comment by rleongcs** [comment]
You just need to configure them under Sensor 1 and 2. Sensor 3 is for top right corner.

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/61628312/b09108da-1f86-47a4-a7c7-6078626eb545)

**Reply by lloydbayley** [reply]
Yep, I got it wrong. Sorry, I meant for the outdoor one where the temperature is beside it on the EU panel and under it on the US panel. Although, space is an issue on the EU one. I'll have to make do I suppose. Thanks for the reply!

**Reply by edwardtfn** [reply]
I have a custom TFT for eu version where I focus on weather. In my case I'm using real feel temperature below the outside temperature, however you can set that to humidity.
Take a look if you want: https://github.com/edwardtfn/HomeAssistant-Config/tree/main/www/nspanel/hall

**Reply by lloydbayley** [reply]
Thank you! Very kind of you. That was my next step - customising my own. :) 
I will indeed have a look at yours.

**Reply by rleongcs** [reply]
@edwardtfn Just need to upload the TFT only right? The rest is using back the same? Do you have a picture of how it looks like?

**Reply by edwardtfn** [reply]
> @edwardtfn Just need to upload the TFT only right?

Exactly.

**Reply by rleongcs** [reply]
Just tried to update the TFT file and all is successful. Unfortunately after the reboot which shows that the TFT is v3.2.3 while my ESPHome and Blueprint is Dev version respectively 3.3.1 and 3.3, the screen just stuck at "Loading". Even a reboot also still stuck at "Loading".

Any suggestion?

**Reply by edwardtfn** [reply]
You need TFT v3.3.1. It is in `dev`, just like blueprint v3.3.1.
And if you install that, please try the new climate page and give us some feedback. 😉

**Reply by rleongcs** [reply]
> I have a custom TFT for eu version where I focus on weather. In my case I'm using real feel temperature below the outside temperature, however you can set that to humidity. Take a look if you want: https://github.com/edwardtfn/HomeAssistant-Config/tree/main/www/nspanel/hall

My feedback was regarding the above. I wanted to try the custom TFT that you have, Thinking about it yesterday, not sure if it was due to the custom date format I have that could be the issue. Will try again later.

I will definitely be trying out the new `dev `build and provide feedback. 😄

**Reply by edwardtfn** [reply]
There are still a few bugs in `dev`. As soon those bugs are gone I will make my customizations. Hopefully I can get there during this weekend.

**Comment by pbranly** [comment]
Hi 
Is the US version ( older of 2 days compared with the EU one) ok for test as well in 3.3.1 ?
Phil

**Reply by edwardtfn** [reply]
The US version currently in DEV doesn't have the new climate page, so it might fail if you try to open that. I think everything else should work fine.

**Reply by pbranly** [reply]
Ok thanks I will wait
